README.rst

```diff
@@ -4,6 +4,15 @@ pytoport
 A fairly normal way to generate FreeBSD port Makefiles straight from
 PyPI.
 
+Requirements
+------------
+
+In order to use pytoport, you should install the required dependencies:
+
+::
+
+    $ pip install -r requirements.txt
```

Comment:
Yes. I'll change this to make it clear that on FreeBSD you should install from ports.